src/utils/bzmesh.jl

```diff
@@ -38,7 +38,10 @@ function bzmesh_ir_wedge(kgrid_size, lattice, composition...; tol_symmetry=1e-5)
     # Ask spglib for symmetry operations and for irreducible mesh
     spg_symops = spglib.get_symmetry(spglib_cell(lattice, composition...),
                                      symprec=tol_symmetry)
-    spg_symops !== nothing || error("Could not get symmetry")
+
+    # If spglib does not find symmetries, fall back to regular grid
+    spg_symops === nothing && return bzmesh_uniform(kgrid_size)
+
```

Comment:
I'm not sure this is such a good idea, actually. Something like this should not go unnoticed. At least one should issue a warning, but I'm not even sure about that. This happens pretty early in a typical calculation, so I think it's fine to just kill everything and let the user decide what to do. When I'm working with DFTK this issue usually arises when I'm stupid with the lattice, so it's been good to force me to go back and reconsider.

Reply:
Why? It's perfectly fine to do a computation with no specific symmetries. It happens for quasi-1D.

Reply:
Hm, come to think of it it should at least have time-reversal symmetry; or is it taken care of somewhere else?

Reply:
Nope, nowhere. `bzmesh_uniform` is without any symmetry taken into account.

Reply:
> Why? It's perfectly fine to do a computation with no specific symmetries. It happens for quasi-1D.

Sure, but `spglib` can fail for different reasons, too, like when you accidentally transpose the lattice matrix. :smile: ... this happens quite quickly when intertwining Julia and python.

Reply:
What do you mean fails? If you pass it the transpose then it just finds a different symmetry group, no?

Reply:
OK so you're saying that even in the purely non-symmetric case it should at least have the time-reversal symmetry in? So when it returns nothing it means some kind of internal error?

Reply:
It is not so clear ... I checked the code. Essentially it can return nothing, when two atoms are too close, when the spacegroup search failed, if it does not find symmetry operations and a few other things ... so some are internal, some are due to issues with the lattice setup. There is some sort of more detailed error message you can get, but I am not sure it is always set (it's a very c-like interface).

I think the options we have is either abort and tell the user, or warn the user and fall back to no symmetry.